Jax/Page/ModControls.php

```diff
@@ -386,7 +385,7 @@ private function ipTools(): void
                     onclick="this.form.submitButton=this" value="Ban" />
                 HTML;
 
-            $torDate = gmdate('Y-m-d', strtotime('-2 days'));
+            $torDate = gmdate('Y-m-d', Carbon::parse('-2 days')->getTimestamp());
```

Comment:
I bet with carbon you can go back 2 days instead of parsing the string, probably subDays